File: packages/svg-icons/src-cs/Telerik.SvgIcons/Icons/ChevronLeft.cs
namespace Telerik.SvgIcons
{
    public class ChevronLeft : SvgIconBase
    {
        public ChevronLeft()
        {
            Name = "chevron-left";
            Content = "<path d=\"M15.67 19.67a.607.607 0 1 1-.86.858l-8.1-8.1a.607.607 0 0 1 0-.859l8.1-8.1a.607.607 0 0 1 .86.859L8 11.999z\"/>";
            ViewBox = "0 0 24 24";
            Variants = new System.Collections.Generic.Dictionary<string, string>
            {
                { "solid", "<path d=\"M16.05 3.9v16.2a.81.81 0 0 1-1.384.574l-8.1-8.1a.81.81 0 0 1 0-1.146l8.1-8.1a.81.81 0 0 1 1.384.573z\"/>" },
                { "outline", "<path d=\"M15.67 19.67a.607.607 0 1 1-.86.858l-8.1-8.1a.607.607 0 0 1 0-.859l8.1-8.1a.607.607 0 0 1 .86.859L8 11.999z\"/>" },
                { "duotone", "<path d=\"M15.24 3.9v16.2L7.14 12z\" opacity=\"0.2\"/><path d=\"M15.55 3.151a.81.81 0 0 0-.884.175l-8.1 8.1a.81.81 0 0 0 0 1.146l8.1 8.1a.81.81 0 0 0 1.384-.573v-16.2a.81.81 0 0 0-.5-.749zm-1.12 14.993-6.145-6.145 6.145-6.145z\"/>" }
            };
        }
    }
}

File: packages/svg-icons/src/icons/chevron-left.ts
import { SVGIcon } from '../svg-icon.interface';

export const chevronLeftIcon: SVGIcon = {
    name: 'chevron-left',
    content: '<path stroke-linecap="round" stroke-linejoin="round" d="M15 19.5 7.5 12 15 4.5" fill="none"/>',
    viewBox: '0 0 24 24',
    variants: {
        'solid': '<path d="M15.2858 3.8104a.75.75 0 0 0-.8175.1622l-7.5 7.5a.75.75 0 0 0-.2199.5306.75.75 0 0 0 .22.5306l7.4999 7.4999a.75.75 0 0 0 .8177.1629.75.75 0 0 0 .4629-.6935V4.5032a.75.75 0 0 0-.4631-.6928"/>',
        'outline': '<path stroke-linecap="round" stroke-linejoin="round" d="M15 19.5 7.5 12 15 4.5" fill="none"/>',
        'duotone': '<path fill-opacity="0.2" d="M15 19.5 7.5 12 15 4.5z"/><path stroke-linecap="round" stroke-linejoin="round" d="M15 19.5 7.5 12 15 4.5z" fill="none"/>'
    },
    tags: ['chevron', 'left', 'navigation', 'arrows', 'arrow', 'direction', 'caret']
}
